searchcore/src/vespa/searchcore/proton/reference/i_gid_to_lid_change_listener.h

```diff
@@ -16,9 +16,10 @@ namespace proton {
 class IGidToLidChangeListener
 {
 public:
+    using Context = std::shared_ptr<search::IDestructorCallback>;
```

Comment:
I would prefer "DestructorCallbackSP" (or similar) instead of "Context", which is very generic. Then it is possible to look at a function signature and better understand what the first parameter is representing.

Reply:
Fixed in #13828